install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -295,12 +295,15 @@ data:
   # container image names
   sidecar-istio-proxy-image: "{{ .Values.proxy.sidecarImageRegex }}"
 
+{{- if not .Values.gke.enabled }}
```

Comment:
nit: this can be moved to else block of `if` statement at line 317, so that it's more visable going forward.

Reply:
done :smiley:

Reply:
@sayboras  Now that we are setting `tunnel` in the if-else block, should we take the comments from line 313-317 out of the else block, so that it's in the file irrespective of the value set for `.Values.gke.enabled`??
https://github.com/cilium/cilium/blob/527d817e2609cecc7077ad0c50a4be894793336b/install/kubernetes/cilium/templates/cilium-configmap.yaml#L307-L319

Reply:
Agreed, I don't think the comment here provides any value at all. Normally, end-user will check `.values.yaml` or README.md to get more information.

If it's too much trouble for you, happy to leave as what it is as well.